internal/engine/actions/actions.go

```diff
@@ -115,7 +115,7 @@ func (rae *RuleActionsEngine) DoActions(
 		cmd := shouldRemediate(params.GetEvalStatusFromDb(), params.GetEvalErr())
 		// Run remediation
 		result.RemediateMeta, result.RemediateErr = rae.processAction(ctx, remediate.ActionType, cmd, ent, params,
-			nil)
+			getMeta(params.GetEvalStatusFromDb().RemMetadata))
```

Comment:
since we always pass metadata for both remediate and alerts, would it make sense to change the interface for `actions.Do` to accept a `json.RawMessage`, not a pointer to it to avoid crashes in some corner cases?

Reply:
(not in this PR)

Reply:
good point, we certainly can 👍